package.json

```
@@ -19,7 +19,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "lint": "eslint src/**",
+    "lint": "eslint src/**.ts",
```

To ensure subfolders are listed, change to:
```suggestion
    "lint": "eslint src/**/*.ts",
```